test/CMakeLists.txt

```diff
@@ -80,6 +80,11 @@ if(MSVC)
 	# Disable warning C4566: character represented by universal-character-name '\uFF01' cannot be represented in the current code page (1252)
 	# Disable warning C4996: 'nlohmann::basic_json<std::map,std::vector,std::string,bool,int64_t,uint64_t,double,std::allocator,nlohmann::adl_serializer>::operator <<': was declared deprecated
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4389 /wd4309 /wd4566 /wd4996")
+
+    if(MSVC_VERSION GREATER_EQUAL 1910)
+        # Enable c++17 support in Visual Studio 2017 (for testing perfect forwarding and transparent comparator in find() / count())
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /std:c++17")
+    endif()
```

Comment:
IIRC Appveyor already runs with VS2017, you might want to put that flag in the Appveyor file?

Reply:
Indeed we have three configurations for AppVeyor:

- MSVC 2015
- MSVC 2017
- MSVC 2017 + `/permissive- /std:c++latest /utf-8`

Reply:
Ah nice, didn't realize that thanks. I'll this remove this then.